frontend/src/tests/ddah-tests.js

```diff
@@ -88,9 +88,22 @@ export function ddahTests(api) {
         expect(resp1.payload).toContainObject(ddah);
     });
 
-    it.todo(
-        "getting ddahs for one session will not return ddahs for another session"
-    );
+    it("getting ddahs for one session will not return ddahs for another session", async () => {
+        // retrieve all ddahs for the session
+        resp1 = await apiGET(`/admin/sessions/${session.id}/ddahs`);
+        expect(resp1).toHaveStatus("success");
+        const allDdah = resp1.payload;
+
+        // retrieve all the assignments for the seesion
+        const resp2 = await apiGET(`/admin/sessions/${session.id}/assignments`);
+        expect(resp2).toHaveStatus("success");
+        const allAssignmentId = resp2.payload.map((y) => y.id);
+
+        // check that all the ddahs are associated with an assignment from the session
+        allDdah.forEach((d) => {
+            expect(allAssignmentId).toContain(d.assignment_id);
+        });
```

Comment:
The default seed data only has one session and so only puts DDAHs in for that session. You need to create another session, a new assignment for that session, and a DDAH for that assignment. Then make sure fetching the DDAHs for the seed session and the newly created session return different objects.